answering.py

```diff
@@ -0,0 +1,76 @@
+import re
+import json
+from container.answer_loading import load_question_data
+
+
+def parseAnswerTextSpacing(s):
+    # Lowercase version
+    words = re.sub("-", " ", s.lower().strip())
+    words = re.sub("\n+", "\n", words)
+    words = re.sub(" +", " ", words)
+    words = " ".join([w.strip() for w in words.split(" ") if w.strip()])
+    words = words.strip()
+    return words
+
+
+def checkAnsweredQuestions(pageQuestion, answeredQuestions):
+        for answeredQuestion in answeredQuestions:
+            if answeredQuestion.find(pageQuestion) == 0:
+                # and/or other specifications
+                return answeredQuestion
+        return None
+
+
+#### placeholder for data retrieval/recieving endpoint
+def load_question_data_plc(id):
+    import numpy as np
+    import pandas as pd
+    questionData = {}
+    df = pd.read_csv(f"data/"+id+".txt")
+    mtx = np.array(df)
+    for i in range(len(mtx)):
+        row = [ij for ij in list(mtx[i,:]) if str(ij).lower() != "nan"]
+        questionData[row[0]] = []
+        for j in range(1, len(row)):
+            questionData[row[0]] += [row[j]]
+    return questionData
+
+
+def answer_input_questions(id, inputQuestions):
+    
+    ###########
+    # OVERRIDE
+    questionData = load_question_data_plc(id)
+    ##########
+
+    if not questionData:
+        return "No information found for id " + id
+
+    def getAnsweredQuestions(questionData):
+        # points to the input question to retrieve it
```

Comment:
Use triple quotes as docstrings. This way, your IDE will recognize the hints. For example:

```
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """
```

Reply:
Let me know if the version should be finalized after you've tested it out so I can finalize the documentation on main.